Behavioral/Iterator/BookList.php

```diff
@@ -13,7 +12,7 @@ public function getBook($bookNumberToGet)
             return $this->books[$bookNumberToGet];
         }
 
-        return null;
+        return;
```

Comment:
in PHP, this is equal, as there is null returned when no explicit value was given.